### DISCUSSION TITLE
Errors on building in CI

### DISCUSSION THREAD

**Discussion by LekoArts** [napi-rs/napi-rs#1082] [Q&A] [answered]
Hi!
Thanks for creating and maintaining napi-rs, I really enjoy working with it :)

I've been using it for the first time on this PR: https://github.com/LekoArts/thanks-contributors/pull/2
I set up the project with `napi new` explained here: https://napi.rs/docs/introduction/getting-started

Now I'm a bit confused on the failures in the GitHub actions and wondered if I set up something incorrectly.

Two examples:

https://github.com/LekoArts/thanks-contributors/runs/5372351507?check_suite_focus=true

```shell
Create Artifact Container - Attempt 5 of 5 failed with error: connect ETIMEDOUT 13.107.42.16:443
Error: Create Artifact Container failed: connect ETIMEDOUT 13.107.42.16:443
```

https://github.com/LekoArts/thanks-contributors/runs/5372351609?check_suite_focus=true

```shell
error: failed to run custom build command for `openssl-sys v0.9.72`
Error: failed to run custom build command for `openssl-sys v0.9.72`
```

Any idea what those things might be? Thanks a lot!

**Comment by Brooooooklyn** [comment, marked as answer]
@LekoArts try to add `native-tls-vendored` feature in `reqwest` https://github.com/seanmonstar/reqwest/blob/master/Cargo.toml#L38

**Reply by LekoArts** [reply]
Thanks for the quick answer, appreciate it. I've added the feature flag in a new commit and now only two builds fail with this openssl error. Do I need to enable more maybe?

**Reply by Brooooooklyn** [reply]
Seems like `vendored` features in `openssl-sys` need `perl` installed. Add `apk install perl` before `yarn build` in `linux musl` builds

**Reply by LekoArts** [reply]
Thanks, that seems to have done the trick! I realize this was more a personal support request now than a bug in the napi-rs package, appreciate the help!
